Assets/Scripts/Models/Buildable/Components/GasConnection.cs

```diff
@@ -108,7 +109,7 @@ public GasInfo()
             
             public override string ToString()
             {
-                return string.Format("gas:{0}, rate:{1}, min:{2}, max:{3}", Gas, Rate, MinLimit, MaxLimit);
+                return LocalizationTable.GetLocalization("gas_info", Gas, Rate, MinLimit, MaxLimit);
```

Comment:
components should be visible to outside world, and if there is no usage of ToString in GetDescription(), then it's really just for debugging purpose